pocs/tests/test_camera.py

```diff
@@ -114,8 +116,30 @@ def test_create_cameras_from_config_fail(config):
         'model': 'foobar'
     }
 
-    with pytest.raises(error.PanError):
-        create_cameras_from_config(config, simulator=simulator)
+    cameras = create_cameras_from_config(config, simulator=simulator)
+    assert len(cameras) != 2
+
+    # SBIGs require a serial_number, not port
+    config['cameras']['devices'][0] = {
+        'port': '/dev/ttyFAKE',
+        'model': 'sbig'
```

Comment:
I'd like to suggest using the SDK Camera simulator here instead of the sbig model for the same reason as I suggested `simulator` over `canon_gphoto2` below. Not such a simple change in this case, though, because that simulator class doesn't follow the `pocs.camera.<model name>.Camera` convention that `create_cameras_from_config` relies on. It's currently `pocs.camera.simulator.SDKCamera`, but should probably be moved so that it can be created from a config.

Reply:
I think I have taken care of this properly now via the splitting of the files as we discussed IRL.